frontend/src/components/common/ProfileSidebar.vue

```diff
@@ -1,10 +1,9 @@
 <script setup lang="ts">
 import authService from '@/services/AuthService';
+import Close from '@carbon/icons-vue/es/close/16';
+import UserFollow from '@carbon/icons-vue/es/user--follow/16';
```

Comment:
I am not sure if I misunderstand how to use our Icon.vue component, but same comment as in NotificationMessage.